blocks/library/freeform/editor.scss

```diff
@@ -3,11 +3,13 @@
 	margin: -4px;
 	padding: 4px;
 
-	p {
+	p,
+	li {
```

Comment:
Would it work to add the same here as the rich text component?

```scss
&.mce-content-body {
	line-height: $editor-line-height;
}
```

Reply:
I think it needs the -mce-content-body for specificity, but could the code be optimized? I may be missing something.